packaging/functions.sh

```diff
@@ -103,12 +102,12 @@ install_windows_launcher() (
 
 	SRC_PATH="${1}"
 	DEST_PATH="${2}"
-	TARGETPLATFORM="${3}"
-	MOD_ID="${4}"
-	LAUNCHER_NAME="${5}"
-	MOD_NAME="${6}"
-	FAQ_URL="${7}"
-	VERSION="${8}"
+	MOD_ID="${3}"
+	LAUNCHER_NAME="${4}"
+	MOD_NAME="${5}"
+	FAQ_URL="${6}"
+	VERSION="${7}"
+	TARGETPLATFORM="win-x64"
```

Comment:
This is only used twice (and one of those will hopefully be removed soon) - it would be cleaner to inline win-x64 directly instead.